sns/templates/notifications.json.tpl

```diff
@@ -0,0 +1,42 @@
+{
+  "Version": "2012-10-17",
+  "Id": "default_policy",
+  "Statement": [
+    {
+      "Sid": "default_statement",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
```

Comment:
Does this need to be a wildcard, or can it be more specific?

Reply:
It's a wildcard because the actual restrictions are carried out by the `Condition` so it's not actually allowing access to everything. I don't think there's a way to restrict access to a source account using the `Principal` which is why it's done this way.